rfc/rfc-0000-submitted-status-in-resources.md

```diff
@@ -0,0 +1,142 @@
+stamping of an issue, this condition lets the user know about the _first resource_ where an issue id encountered. We
+should present issues per-resource, in the Owner's `status.resources[].conditions` as well.
+
+# Motivation
+
+[motivation]: #motivation
+
+Top level conditions make it difficult to present a list of issues, and we now have a better UX architecture in Owners
+to display per-resource concerns. Users want to see, on a resource-by-resource basis, what the state of the supplychain
+or delivery is.
+
+We expect automation and graphical interfaces to present the status of resources just by querying the workload, and for
+text users (kubectl extreme!) to see issues at a glance.
+
+# What it is
+
+[what-it-is]: #what-it-is
+
+We add or augment `owner.status.resources[].conditions` to contain a `Submitted` condition such as:
+
+```yaml
+apiVersion: carto.run/v1alpha1
+kind: Workload
+metadata:
+  name: my-workload
+    namespace: default
+spec: { ... }
+status:
+  supplyChainRef:
+    kind: ClusterSupplyChain
+    name: supply-chain
+  conditions:
+    - type: Ready
+      status: Unknown
+      reason: MissingValueAtPath
+      message: waiting to read value [.status.latestImage] from resource [image.kpack.io/testing-sc]
+        in namespace [default]
+      lastTransitionTime: "2022-03-22T20:15:21Z"
+    - type: SupplyChainReady
+      status: "True"
+      reason: Ready
+      message: ""
+      lastTransitionTime: "2022-03-22T20:15:05Z"
+    - type: ResourcesSubmitted
+      status: Unknown
+      reason: MissingValueAtPath
+      message: waiting to read value [.status.latestImage] from resource [image.kpack.io/testing-sc]
+        in namespace [default].
+      lastTransitionTime: "2022-03-22T20:15:21Z"
+  resources:
+    - name: my-resource
+      conditions:
+        - type: Ready
+          status: Unknown
+          reason: MissingValueAtPath
+        - type: Submitted     # <<=== this is the new section 
+          status: Unknown
+          reason: MissingValueAtPath
+          message: waiting to read value [.status.latestImage] from resource [image.kpack.io/testing-sc] in namespace [default].
+    - name: another-resource
+      conditions:
+        - type: Ready
+          status: Unknown
+          reason: MissingValueAtPath
+        - type: Submitted     # One for each resource in spec.resources 
+          status: Unknown
+          reason: MissingValueAtPath
+          message: ...
+```
+
+# How it Works
+
+[how-it-works]: #how-it-works
+
+During the creation of all stamped objects, the realizer provides a conditions collection for each resource. This
+includes any existing conditions (
+see [Resource Status on Workload RFC](https://github.com/vmware-tanzu/cartographer/blob/rfc-resources-report-status/rfc/rfc-0000-allow-resources-to-report-status.md))
+and a calculated `type:Ready` condition.
+
+Each `type:Submitted` condition follows the same implementation as the top level `type:ResourcesSubmitted` condition.
```

Comment:
Is the desire to keep the existing top-level `ResourcesSubmitted` condition? With its status aggregated from all of the individual resource level `Submitted` conditions?

It's interesting to have a couple different dimensions of condition aggregation.

Reply:
I hadn't planned to change it's current spec, which is not aggregated, only "first occurrence"